codestodians/food-basket-front-end
# [Feature: #163944209] Re-activate cart component

#### What does this PR do?
- Adds the Cart component to the existing cart icon

#### Description of Task to be completed?
- Add cart related components
- Ensure the display is appropriate.
- Make it possible to add and remove items ui-wise

#### How should this be manually tested?
- Check out into the `feature/activate-cart-component` branch.
- Start the app and click on the cart icon

#### Any background context you want to provide?
- The cart functionality will be dependent on the actual products, and that will be plugged in subsequently when the products page is ready.

#### What are the relevant pivotal tracker stories?
- [#163944209](https://www.pivotaltracker.com/story/show/163944209)

#### Screenshots
- 
<img width="1680" alt="screen shot 2019-02-13 at 6 31 09 pm" src="https://user-images.githubusercontent.com/22825700/52731629-6ec1d800-2fbe-11e9-8eab-8c16b1ddf8c2.png">
